src/validate.js

```diff
@@ -152,10 +153,12 @@ export default function validateFormData(
   formData,
   schema,
   customValidate,
-  transformErrors
+  transformErrors,
+  local
 ) {
   try {
     ajv.validate(schema, formData);
+    localize[local] && localize[local](ajv.errors);
```

Comment:
All avji18n will be imported. Why, insted of take a "string" in props, taking a function, that can perform the localization ?
The user (dev?) will can import only needed local function, and pass it. 

```suggestion
    localize && localize(ajv.errors);
```

Reply:
I agree